### ISSUE TITLE
Can we work on why there's the pop or hiss in some generate voice samples

### ISSUE THREAD

**Issue by teamblubee** [yl4579/StyleTTS2#194]
The title says a lot but the issue is sometimes there's the random noise at the end of a generated sample or sometimes there's a metallic sound in the middle of a voice sample.

I have noticed that if a deep breathe is made there will be a metallic hiss, to me it seems as if there's no valid voice sample for the sound the algorithm falls back to a  metallic sound similar to the first stage training to generate the mel spectrum.

Is there a way to have some debugging where we can see where in the data was sampled to generate the audio, this could help us debug if these metallic sounds even on well trained voice models be an issue of insufficient data to generate quality output.

**Comment by teamblubee** [comment]
@akaihaoshuai can you share the hiss you're talking about? I'll get some of the hiss that I have and try to upload it some where to share it here as well.

**Comment by danablend** [comment]
Hello!

They already solved this issue here:
https://github.com/yl4579/StyleTTS2/discussions/81

TL;DR >> Add a little bit of silence at the beginning and end of each training sample in your dataset.

**Comment by akaihaoshuai** [comment]
> @akaihaoshuai can you share the hiss you're talking about? I'll get some of the hiss that I have and try to upload it some where to share it here as well.

I solved it by modifying the configuration file as follows


config = yaml.safe_load(open("models/LJSpeech/config.yml"))
